### PR TITLE
New package: Taiizor.SucroseWallpaperEngine version 24.7.13.0

### PR DESCRIPTION
**Pull request by Taiizor**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162866)

### CODE REVIEW

**Comment by Taiizor** [comment]
@microsoft-github-policy-service agree

File: manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+PackageIdentifier: Taiizor.SucroseWallpaperEngine
+PackageVersion: 24.7.13.0
+PackageLocale: en-GB
+Publisher: Taiizor
+PublisherUrl: https://github.com/Taiizor
+PublisherSupportUrl: https://github.com/Taiizor
+PrivacyUrl: https://github.com/Taiizor/Sucrose/blob/develop/PRIVACY_POLICY.md
+Author: Taiizor
+PackageName: Sucrose Wallpaper Engine
+PackageUrl: https://github.com/Taiizor/Sucrose
```
Comment (Trenly) [inline manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.locale.en-GB.yaml:13]: From the package's GitHub, they link directly to the Microsoft Store. This means the application can already be installed using `winget install xp8jgpbhtjglcq` 
```
PS C:\Users\Trenly> winget search "Sucrose"
Name                     Id             Version Source
--------------------------------------------------------
Sucrose Wallpaper Engine XP8JGPBHTJGLCQ Unknown msstore
```

image

File: manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  Silent: /s
+  SilentWithProgress: /s
+Installers:
+- InstallerUrl: https://www.vegalya.com/Sucrose/x64/Sucrose_Bundle_.NET_Framework_4.8_x64_24.7.13.0.exe
+  Architecture: x64
+  InstallerSha256: 4C06F9D224B08CB091CFE4CD9B63AE7FC38C7972AA3DA5F6EB43BCB40C16879E
+- InstallerUrl: https://www.vegalya.com/Sucrose/x86/Sucrose_Bundle_.NET_Framework_4.8_x86_24.7.13.0.exe
+  Architecture: x86
+  InstallerSha256: F6FBC980DFB13E439F01B5E324D564655A925DEA133EF5CEF7206EA5F51E2B2A
+- InstallerUrl: https://www.vegalya.com/Sucrose/ARM64/Sucrose_Bundle_.NET_Framework_4.8_ARM64_24.7.13.0.exe
```
Comment (stephengillie) [inline manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.installer.yaml:17]:
```suggestion
- InstallerUrl: https://github.com/Taiizor/Sucrose/releases/download/v24.7.13.0/Sucrose_Bundle_.NET_Framework_4.8_x64_24.7.13.0.exe
  Architecture: x64
  InstallerSha256: 4C06F9D224B08CB091CFE4CD9B63AE7FC38C7972AA3DA5F6EB43BCB40C16879E
- InstallerUrl: hhttps://github.com/Taiizor/Sucrose/releases/download/v24.7.13.0/Sucrose_Bundle_.NET_Framework_4.8_x86_24.7.13.0.exe
  Architecture: x86
  InstallerSha256: F6FBC980DFB13E439F01B5E324D564655A925DEA133EF5CEF7206EA5F51E2B2A
- InstallerUrl: https://github.com/Taiizor/Sucrose/releases/download/v24.7.13.0/Sucrose_Bundle_.NET_Framework_4.8_ARM64_24.7.13.0.exe
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Taiizor,

The `PackageUrl` doesn't really lead to the current `InstallerUrl`s, but does lead to these URLs. Should they be used instead?

I saw the "Sponsor this project" link, but couldn't navigate through it to the `InstallerUrl`s. Is there another way to navigate from one to the other in a browser?

File: manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.locale.en-GB.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.locale.en-GB.yaml:13]: Should this be a blocker? I believe other packages can be installed through both sources.

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**